device/bgp/bgpupdatereplay.yaml

```diff
@@ -0,0 +1,33 @@
+components:
+  schemas:
+    Bgp.UpdateReplay:
+      description: >-
+        Ordered BGP Updates ( including both Advertise and Withdraws ) to be sent in the order given in the input to the peer after the BGP session is established.
+      type: object
+      properties:
+        updates:
+          description: >-
+            Array of ordered BGP Updates ( including both Advertise and Withdraws ) to be sent in the order given in the input to the peer after the BGP session is established. 
+          type: array
+          items:
+            $ref: '#/components/schemas/Bgp.OneUpdateReplay'
+          x-field-uid: 1
+    Bgp.OneUpdateReplay:
+      description: >-
+        Specification of one BGP Update to be sent to the BGP peer.        
+      type: object
+      properties:
+        update_bytes:    
+          description: >-
+            Bytes specified in hex format to be sent to peer after the BGP Update Header. The Update Header will always have the 
+            initial 16 bytes containing Marker bytes, 2 bytes containing the Length and 1 byte containing the Type.The string MUST 
+            contain sequence of valid hex bytes.The bytes specified in hex format should be appended to the Update message to be sent 
```

Comment:
minor nit: space is missing before the new line.

Reply:
Done.